lib/chef_zero/server.rb

```diff
@@ -164,10 +164,10 @@ def port
     #
     def url
       sch = @options[:ssl] ? 'https' : 'http'
-      @url ||= if @options[:host].include?(':')
-                 URI("#{sch}://[#{@options[:host]}]:#{port}").to_s
+      @url ||= if @options[:host].first.include?(':')
+                 URI("#{sch}://[#{@options[:host].first}]:#{port}").to_s
                else
-                 URI("#{sch}://#{@options[:host]}:#{port}").to_s
+                 URI("#{sch}://#{@options[:host].first}:#{port}").to_s
```

Comment:
I'm conflicted about that. On the one hand, that's clearly not the right behavior. On the other hand the use case for zero is often just to serve a repo of existing files, when you won't actually trigger said incorrect behavior.